sections/preliminaries.tex

```diff
@@ -7,4 +7,15 @@ \section{Preliminaries}
 Finally, the vector of attributes for the \textit{i}-th sensor at timestamp \textit{t} is represented with \(x_{t}^{(i)} \in \mathbb{R}^{C}\). \cite{cirsteaEnhanceNetPluginNeural2021}
 
 % maybe TODO: Define correlated time series forecasting, if we end up using it.
-% TODO: Formal problem definition 
\ No newline at end of file
+% TODO: Formal problem definition 
+
+\subsection{Problem overview}
+Given a time series as aforementioned, it is useful to be able to forecast the future behavior of the time series.
+This can be done by using a model that is able to predict the future behavior of the time series.
```

Comment:
```suggestion
This can be done using a learning model that is able to predict that future behavior.
```

Reply:
Not sure statistical is the right word here. ARIMA is the only stats model we use.

Reply:
```suggestion
This can be done using a model that is able to predict that future behavior.
```